### PR TITLE
make prepare script work on windows

### PR DESCRIPTION
**Pull request by frederikhors**
Fix https://github.com/FormidableLabs/urql/issues/744.

### CODE REVIEW

File: packages/svelte-urql/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "check": "tsc --noEmit",
     "lint": "eslint --ext=js,jsx,ts,tsx .",
     "build": "rollup -c ../../scripts/rollup/config.js",
-    "prepare": "../../scripts/prepare/index.js",
+    "prepare": "node ../../scripts/prepare/index.js",
```
Comment (kitten) [inline packages/svelte-urql/package.json:46]: Thank for fixing this. Must’ve been an oversight as we only test on Windows occasionally.
Can you update this in all package.json files please, if it’s not too much trouble? 🙌

Comment (frederikhors) [reply packages/svelte-urql/package.json:46]: Sure. One minute.

**Comment by frederikhors** [comment]
Done.

**Comment by kitten** [comment]
@frederikhors Sorry, me again, 😅  There are more packages located in `exchanges/*` so all `exchanges/*/package.json` files are affected as well.

**Comment by frederikhors** [comment]
Done! 😄

**Comment by kitten** [comment]
@frederikhors I pushed the above fix for #744 to your branch. Can you test out whether that fixes the path compare issue for you please?

**Comment by frederikhors** [comment]
```
> @urql/svelte@0.2.1 prepare C:\projects\svelte\urql\urql\packages\svelte-urql
> node ../../scripts/prepare/index.js

C:\projects\svelte\urql\urql\scripts\prepare\index.js:21
  path.normalize(a).replace(new RegExp(path.sep, 'g'), '/')
                            ^

SyntaxError: Invalid regular expression: /\/: \ at end of pattern
    at new RegExp (<anonymous>)
    at is (C:\projects\svelte\urql\urql\scripts\prepare\index.js:21:29)
    at Object.<anonymous> (C:\projects\svelte\urql\urql\scripts\prepare\index.js:32:4)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
```

**Comment by frederikhors** [comment]
It works now!

**Pull request merged**